linkis-commons/linkis-common/src/main/java/org/apache/linkis/common/io/FsPath.java

```diff
@@ -169,8 +167,8 @@ public File toFile() {
     return new File(uri);
   }
 
-  public Path toPath() {
-    return FileSystems.getDefault().getPath(uri.toString());
+  public String getUriString() {
```

Comment:
A modification of this method requires the addition of a single test

Reply:
Thanks for your advice, I would add unit test for FsPath class.